ocs_ci/ocs/platform_nodes.py

```diff
@@ -1,24 +1,37 @@
+import json
 import logging
 import os
-import json
 import re
+import shutil
+import time
+
 
 import boto3
 import yaml
 
+from ocs_ci.deployment.terraform import Terraform
+from ocs_ci.deployment.vmware import (
+    change_vm_root_disk_size,
+    clone_openshift_installer,
+)
 from ocs_ci.ocs.exceptions import TimeoutExpiredError
 from ocs_ci.framework import config, merge_dict
 from ocs_ci.utility import aws, vsphere, templating
 from ocs_ci.utility.retry import retry
 from ocs_ci.utility.csr import approve_pending_csr
 from ocs_ci.ocs import constants, ocp, exceptions
-from ocs_ci.ocs.node import get_node_objs, get_typed_worker_nodes
+from ocs_ci.ocs.node import (
+    get_node_objs, get_typed_worker_nodes, get_typed_nodes,
+)
 from ocs_ci.ocs.resources.pvc import get_deviceset_pvs
 from ocs_ci.ocs.resources import pod
+from ocs_ci.utility.csr import (
+    get_nodes_csr, wait_for_all_nodes_csr_and_approve,
+)
 from ocs_ci.utility.utils import (
     get_cluster_name, get_infra_id, create_rhelpod,
-    get_ocp_version, TimeoutSampler,
-    download_file, delete_file, AZInfo
+    get_ocp_version, TimeoutSampler, download_file,
+    delete_file, AZInfo, replace_content_in_file, clone_repo,
```

Comment:
I see travis is failing with clone_repo imported but unused

Reply:
done